src/Sylius/Bundle/CoreBundle/Resources/config/app/fixtures/shop_configuration.yaml

```
@@ -253,3 +253,9 @@ sylius_fixtures:
                                 name: 'Winter sale'
                                 channels:
                                     - 'FASHION_WEB'
+                                        
+                catalog_promotion_rule:
+                    options:
+                        custom:
+                            for_variants:
+                                catalogPromotion: 'winter'
```

imo this config shoudl look like:
```suggestion
                            rule_for_variants:
                                catalogPromotion: 'winter'
                                rule: 'for_variants'
                                configuration:
                                - 'first_variant_code'
                                - 'second_variant_code'
```

WDYT about it :)?